app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
1. You can simplify:
```python3
if vstr.get("vaccine") is None:
```

- Key Checking in dictionaries: Python's dict has a method called get() that returns the value for the given key if it exists or a default value otherwise. This makes it easy to check for the existence of keys.

- Comparisons to singletons like None should always be done with is or is not, never the equality operators.